firewall-config.yaml

```diff
@@ -0,0 +1,78 @@
+---
+imports:
+   - path: https://storage.googleapis.com/rs-gdm-stl-master/constants_v1/constants_v1.py
+     name: constants_v1.py
+   - path: https://storage.cloud.google.com/rs-gdm-stl-master/firewall_rule_v1/firewall_rule_v1.py                                                     
```

Comment:
Do not use this template, it adds no value.  Instead use `compute.v1.firewall`.

Reply:
Took it off

Reply:
Disregard his comment, the rs_bastions_firewall_rule_v1.py uses the firewall_v1.py template.